finance/accounting.md

```diff
@@ -1,52 +1,68 @@
 # Accounting data
 
 Accounting data describes the money that we **know** we have spent or that we **highly expect** to receive.
-There are a few kinds of accounting data we have access to:
 
-All 2i2c-specific accounting documents should be accessible on our Team Drive to anybody with a `@2i2c.org` address.
-We don't control the access conditions for the CS&S documents described below, so if you don't have access to these, ask the {role}`Executive Director`.
+## Accounting transactions airtable
 
-We split our financial and accounting data between two places:
+{term}`CS&S` provides us a monthly dataset of the last 6 months of all of all our accounting transactions.
+This is [in a Google Sheet that is automatically updated each day][gsheet].
+It includes the last 6 months of any transaction 2i2c has made (if we need more than 6 months of history, ask `fsp@2i2c.org` for a custom data dump).
 
-- [2i2c Drive Folder with our finance assets](https://drive.google.com/drive/folders/1D5NQKhPDP6zMQ8EdLcMOceTz-ek81nmX?usp=sharing) contains 2i2c-specific budgeting and modeling information. It is available to all team members.
+We have a copy of this sheet in an [**Accounting Transactions AirTable**][accounting-table][^airtable].
+Any columns that are synchronized with CS&S have a little lightning bolt (⚡) next to them.
+We use this to create [our accounting dashboard](accounting:dashboard).
+
+[^airtable]: See [](../administration/airtable.md) for how to access AirTable.
+
+[gsheet]: https://docs.google.com/spreadsheets/d/1qH5IK18z79X8cEwlwDwnlArTiYnbRVIPGqdOUdrsF0c/edit?usp=sharing
+
+[accounting-table]: https://airtable.com/appbjBTRIbgRiElkr/tblDKGQFU0iEIa5Qb/viwAdsIgMwbqKDdZ0
+
+### Update our accounting data airtable
+
+We have a semi-automated process to update the data in our accounting table.
+Here are the steps to follow:
+
+% TODO: We should automate this with a CRON job, make.com/zapier integration, etc.
+%   ref: https://github.com/2i2c-org/team-compass/issues/703
+1. **Confirm that [the Google Sheet][gsheet] has been updated** by checking the `Automatic Operations Events Log` tab.
+2. **Delete all of the records** in the [accounting table][accounting-table].
+3. **Import the google sheets data**. Go to `Accounting Transactions` -> `Import Data` -> `Google Sheets` -> `Google sheets account`.
+   - Here's an image of the menu you want:
+
+     ![image](https://user-images.githubusercontent.com/1839645/230121196-0d398812-ba22-4cea-a42f-e3ad644a3e19.png)
+   - You'll see a list of Google Sheets in our account. Import the one titled `2i2c FYE23 Account Transactions - Auto Generated`.
```

Comment:
The FYE23 title here will become stale after a year. Can we make this process evergreen? Or should we document FY end changes to the procedures?

Reply:
I think this is a question for Joe - he's the one that chose the title. I think it may take a bit of iteration to improve this, can we encapsulate this in a new issue that we can track, similar to  https://github.com/2i2c-org/team-compass/issues/703 ?

Reply:
I agree. I'll add this into #703 as it seems appropriate there.